modules/get-started/pages/rpk-first-steps.adoc

```diff
@@ -0,0 +1,240 @@
+= Quickstart with rpk
```

Comment:
You could consider moving it this page into the Quickstart section in the left nav, and just have links to it in the pages in the Redpanda CLI section.

Reply:
If you do move it there, I'd probably rename it Redpanda CLI Quickstart

Reply:
Or **Redpanda rpk Quickstart**?